client/src/main/python/slipstream/wrappers/BaseWrapper.py

```diff
@@ -22,11 +24,35 @@
 from slipstream.SlipStreamHttpClient import SlipStreamHttpClient
 from slipstream.NodeDecorator import NodeDecorator
 from slipstream.exceptions import Exceptions
+from slipstream.NodeInstance import NodeInstance
+from slipstream.exceptions.Exceptions import TimeoutException, ExecutionException
+
+
+class InconsistentScaleStateError(Exceptions.ExecutionException):
+    pass
+
+
+class InconsistentScalingNodesError(Exceptions.ExecutionException):
```

Comment:
@konstan: 
I think that these two classes should be moved into `Exceptions.py`.
Because this is the place where all SlipStream exceptions are.
What do you think ?